src/main/java/at/hannibal2/skyhanni/features/rift/everywhere/Ubikreminder.kt

```diff
@@ -0,0 +1,37 @@
+package at.hannibal2.skyhanni.features.rift.everywhere
+
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.features.rift.RiftAPI
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.hours
+
+@SkyHanniModule
+object Ubikreminder {
+
+    private val config get() = RiftAPI.config.ubikReminder
+
+    private var isTimerRunning = false
+    private val messageRegex = Regex("ROUND [1-9] \\(FINAL\\):")
+
+
+    @SubscribeEvent
+    fun onChat(event: LorenzChatEvent) {
+        if (!RiftAPI.config.ubikReminder) return
+        val message = event.message
+        if (messageRegex.matches(message) && !isTimerRunning) {
+            startTimer()
+        }
+    }
+    private fun startTimer() {
+        isTimerRunning = true
+
+        DelayedRun.runDelayed(2.hours) { // 2 hours as a Duration
```

Comment:
instead of running a timer, please save the moment in time the player did the interaction, then check every second if the duration is over 2 hours.